File: Frontend/TopConWeb-AngularV8/src/app/classes/grupo-economico/grupo-economico.ts
import { CadastroGeral } from "../cadastro-geral/cadastro-geral";
import { Interveniente } from "./../interveniente/interveniente";

export class GrupoEconomico {
    codigo: number = 0;
    descricao: string = '';
    limiteValor: number = 0.00;
    limiteData: Date;
    bloqueioMotivo: CadastroGeral;
    bloqueioObservacao: string = '';
    clientes: Interveniente[] = [];
    idCadastro: string = '';
    idAtualizacao: string = ''; 
}

File: Backend/TopSys.TopConWeb.Domain/Entities/GrupoEconomico.cs
using System;
using System.Collections.Generic;

namespace TopSys.TopConWeb.Domain.Entities
{
    public class GrupoEconomico
    {
        public int Codigo { get; set; }

        public string Descricao { get; set; }

        public float LimiteValor { get; set; }

        public DateTime? LimiteData { get; set; }

        public int? BloqueioMotivoCodigo { get; set; } = 0;

        public string BloqueioObservacao { get; set; } = "";

        public virtual CadastroGeral BloqueioMotivo { get; set; }

        public string IdCadastro { get; set; }

        public string IdAtualizacao { get; set; }

        public virtual ICollection<Interveniente> Clientes { get; set; }
    }
}
